dev_tools/conf/mypy.ini

```diff
@@ -7,22 +7,21 @@ follow_imports = silent
 ignore_missing_imports = true
 
 # 3rd-party libs for which we don't have stubs
-[mypy-apiclient.*,freezegun.*,matplotlib.*,mpl_toolkits,multiprocessing.dummy,oauth2client.*,pandas.*,proto.*,pytest.*,scipy.*,sortedcontainers.*,setuptools.*,pylatex.*,networkx.*,qiskit.*,pypandoc.*,ply.*,_pytest.*,google.api.*,google.api_core.*,grpc.*,google.auth.*,google.oauth2.*,google.protobuf.text_format.*,quimb.*,pyquil.*,google.cloud.*,filelock.*,codeowners.*,tqdm.*,importlib_metadata.*,google.colab.*,IPython.*,astroid.*,pylint.*]
+
+# Google
+[mypy-google.api_core.*,,google.auth.*,google.colab.*,google.protobuf.text_format.*,google.protobuf.*,google.cloud.*]
```

Comment:
- Please replace `,,` --> `,`
- `google.protobuf.*` should be perhaps removed, it has its own block at line 25